components/WhatsNewDialog.tsx

```
@@ -0,0 +1,328 @@
+        currentEntry = {
+          version: versionMatch[1],
+          date: versionMatch[2],
+          changes: []
+        }
+        currentChangeType = null
+        console.log('Found version:', versionMatch[1], 'date:', versionMatch[2])
+        continue
+      }
+
+      // Match change type headers with emojis like ### 🆕 Nuevas funciones
+      const changeTypeMatch = line.match(/^### [🆕✨🔧❌] (.+)/)
```

Biome (2.1.2)

[error] A character class cannot match a surrogate pair. Add the 'u' unicode flag to match against them.

A surrogate pair forms a single codepoint, but is encoded as a pair of two characters. Without the unicode flag, the regex matches a single surrogate character.
Safe fix: Add unicode u flag to regex


(lint/suspicious/noMisleadingCharacterClass)